src/providers/config/config.ts

```diff
@@ -228,7 +229,7 @@ export class ConfigProvider {
             this.configCache = _.clone(config);
             this.backwardCompatibility();
```

Comment:
function name:
setOldConfig?

Reply:
We could change the name of that function in another pull request